src/newViews/AnalysisModule/Analysis/index.tsx

```diff
@@ -76,6 +77,20 @@ const barChartMargin = {
 const BAR_TICK_COUNT = 5;
 const MAX_BAR_SIZE = 16;
 
+function shouldHideEdit({
+    hasAnalysisFramework,
+    entryPermissions,
+}: {
+    hasAnalysisFramework: boolean;
+    entryPermissions: {
+        create: boolean;
+        modify: boolean;
+    };
+}) {
+    return (!hasAnalysisFramework || !(entryPermissions.create || entryPermissions.modify));
```

Comment:
Why not just use modify?

Reply:
We've duplicated the edit entries permissions here